### PR TITLE
RN: Change field from errorDisplayMessage to displayMessage

### PR DESCRIPTION
**Pull request by amytang0**
This helps us with consistency across all 4 platforms (Web, Android, iOS, and RN)

### CODE REVIEW

File: Types.ts [hunk 1]
```diff
@@ -291,7 +291,9 @@ export interface LinkError {
     errorCode: LinkErrorCode;
     errorType: LinkErrorType;
     errorMessage: string;
+    /** @deprecated Use displayMessage instead */
     errorDisplayMessage?: string;
```
Comment (amytang0) [inline Types.ts:295]: Remove this in the next major update https://jira.plaid.com/browse/RNSDK-109

Comment (pladaub) [reply Types.ts:295]: Just curious: where does the `@deprecated` come from? Is it just something we added, or does it have IDE support in some IDEs?

**Comment by pladaub** [comment]
LGTM – just want to double check that merging onto `main` won't cause this to be pulled through npm or anything, people would have to clone the repo to grab this?

**Comment by amytang0** [comment]
Correct, they'd have to clone the repro

File: Types.ts (hunk 1 above)
Comment (amytang0) [reply Types.ts:295]: https://jsdoc.app/tags-deprecated.html
It's a jsdoc tag

**Review by pladaub — APPROVED** [review]

**Pull request merged**